dbt_semantic_interfaces/parsing/where_filter/parameter_set_factory.py

```diff
@@ -31,7 +32,10 @@ def _exception_message_for_incorrect_format(element_name: str) -> str:
 
     @staticmethod
     def create_time_dimension(
-        time_dimension_name: str, time_granularity_name: Optional[str] = None, entity_path: Sequence[str] = ()
+        time_dimension_name: str,
+        time_granularity_name: Optional[str] = None,
+        entity_path: Sequence[str] = (),
```

Comment:
Do we allow keyword arguments for TimeDimensions? I've gotten errors on that before but not sure if it's changed! I ask because as is, you would be forced to pass something in the `entity_path` field in order to pass a `date_part_name`. So it would be good to fix that! And/or change the order of params here so that date_part comes first (but that might be a breaking change)

Reply:
Ya, we can't change the order of the params. I added support for kwargs here https://linear.app/dbt-labs/issue/SL-1100/multiple-updates-to-query-interface

Reply:
Sweet!!